### ISSUE TITLE
Request second opinion on WP_048029797.1 from PMID:31169899

### ISSUE THREAD

**Issue by danielhhaft** [arpcard/amr_curation#33]
Based on PMID:31169899, NCBI has named proteins similar to WP_048029797.1 as

"Ecr family regulatory small membrane protein".   However, during review of related proteins we determined that the typical length for members of the family is 46 amino acids, not 72 amino acids. This suggests that colistin resistance detected and described by the authors might reflect changes in levels of Ecr expression from mutations upstream of the translation start site, rather than direct activity of an unusual, N-terminally Ecr extended protein variant.  Some homologs of WP_048029797.1 cannot be extended to similar lengths because a stop codon is encountered.

WP_045909000.1 is NCBI's representation of the 46 amino acid form.   

CARD, please take a look and see how you think the ecr findings are best interpreted.

**Comment by danielhhaft** [comment]
The author sent a comment, in response to a query from me.  

"Thank you for the message ... we are fully aware [of shorter homologs] when we performed the analysis. We believe that Ecr is indeed 72 aa in length as predicted from ATG start condon and it is a hybrid containing the last 46-aa (commonly seen in Enterobacter and some other bacterial species) and the N-terminal 26-aa, which is critical to mediate colistin resistance. For Enterobacter strains with only the 46-aa version similar protein, they are susceptible to colistin as we tested for some strains of our Enterobacter collection."

I am sympathetic to their interpretation, but remain hesitant to embrace it as a description of a novel colistin resistance protein.
